README.md

```diff
@@ -34,6 +34,14 @@ execute the script which leverages `util/populate_dev_data.go` by running:
 ./util/docker-dev/dev_data.sh
 ```
 
+*NOTE*: Some web UI views, such as `/interop`, only work in the presence of so-called "static runs". If some views
+do not work while debugging locally, try clearing your local datastore and memcache, then run:
```

Comment:
@lukebjerring this is no longer true with your #487 , right?

Reply:
Yes, the intention is that `populate_dev_data` works locally with or without the static_runs. I've got to do another PR, to enumerate `complete` and not, first before I'll make the claim that we're in that state.

Reply:
Found this old PR lying around. @lukebjerring you've been working on the related code. Is any of this PR still relevant or should I abandon it?